src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: (a) => {
+      calculator.result += a;
```

Comment:
use `this.result`, because you’re in the body of an object, and it’s the best practice